calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -54,6 +54,12 @@ You simply enable WAF in Manager UI, and determine the services that you want to
 
 **Required**
 
+:::note
+
+Do this step only if you [enable WAF by using the CLIs](#enable-waf-using-clis).
```

Comment:
Suggestion. 

**Required**

If you intend to enable WAF using the CLI rather than in Manager UI, or you have not yet [configured L7 logs](), you must enable the Policy Sync API in Felix. To do this cluster-wide, modify the `default` FelixConfiguration to set the field `policySyncPathPrefix` to `/var/run/nodeagent`:

Reply:
Thanks Barbara. Fixed.